File: Famoser.SyncApi/Api/Communication/Response/CollectionEntityResponse.cs
using System.Collections.Generic;
using Famoser.SyncApi.Api.Communication.Entities;
using Famoser.SyncApi.Api.Communication.Response.Base;

namespace Famoser.SyncApi.Api.Communication.Response
{
    public class CollectionEntityResponse : BaseResponse
    {
        public List<CollectionEntity> CollectionEntities { get; set; } = new List<CollectionEntity>();
    }
}

File: Famoser.SyncApi.Webpage/src/Models/Communication/Response/CollectionEntityResponse.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 04.11.2016
 * Time: 19:20
 */

namespace Famoser\SyncApi\Models\Communication\Response;


use Famoser\SyncApi\Models\Communication\Entities\CollectionEntity;
use Famoser\SyncApi\Models\Communication\Response\Base\BaseResponse;

class CollectionEntityResponse extends BaseResponse
{
    /* @var CollectionEntity[] $CollectionEntities */
    public $CollectionEntities;
}
